Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/categories-tree/CategoriesTree.tsx

```diff
@@ -0,0 +1,105 @@
+﻿import React, { useCallback, useEffect, useState } from 'react';
+import { TreeItem, TreeView } from '@material-ui/lab';
+import { isArray, isEmpty, without } from 'lodash';
+import MdExpandMoreIcon from '../../guidelines/icons/MdExpandMoreIcon';
+import MdChevronRightIcon from '../../guidelines/icons/MdChervronRightIcon';
+import ITreeItemType from '../../../common/tree-types';
+import { ITreeProps } from '../../guidelines/trees/Tree';
+
+import styles from './CategoriesTree.module.scss';
+
+interface ICategoriesTreeProps extends ITreeProps{
+}
+
+const CategoryTree = ({
+    items,
+    onCategoryLabelClick,
```

Comment:
The file is named `CategoriesTree`. i.e. plural, but the name of the component is `CategoryTree`, i.e. singular.